### PR TITLE
Add xrtpy and irispy-lmsal to the affiliated package page

### LINKED ISSUE #414: Submit `irispy-lmsal` as an affiliated package
## Package Details

- Package name: irispy-lmsal
- Maintainer(s): Nabil Freij (@nabobalis)
- Repository: https://github.com/LM-SAL/irispy-lmsal 
- Documentation link: https://irispy-lmsal.readthedocs.io/en/stable/ 
- PyPI link (if applicable): https://pypi.org/project/irispy-lmsal/
- Continuous integration link(s): https://github.com/LM-SAL/irispy-lmsal/actions
- Link to Code of Conduct or page describing adopted Code of Conduct: errrrrrrrrrr

## Description of Package

A Python package that provides the tools to read in and analyze data from the IRIS solar-observing satellite.

## Editor Submission Checklist

- [x] Is the submission appropriate (compatible with the SunPy CoC)?
- [x] Does the project have an [appropriate license](https://opensource.org/licenses/)?
- [x] Is the project in an online version control system?
- [x] Does the project provide a Python interface?
- [x] Is the project on PyPI?
- [x] Is the project useful to the solar physics community?
- [x] Version that was reviewed: v0.2.0

## Instructions to Reviewer

Please copy the following and select the ranking for each criteria, the full review criteria can be found here:

```
* Functionality           : General Package / Specialized Package / Not Relevant
* Integration             : Well integrated / Partially Integrated / Minimal Integration
* Documentation           : Extensive / Some / Little
* Testing                 : Excellent / Good / Needs Work
* Duplication             : None / Some / Major
* Community               : Excellent / Good / Needs Work
* Development Status      : Stable / Subject to Change / Low Activity / Needs Work
```

### LINKED ISSUE #431: Affiliated Package Review Request: XRTpy
## Package Details

- Package name: XRTpy
- Maintainer(s): Joy Veasquez  (@joyvelasquez), Nick Murphy (@namurphy), & Jonathan Slavin (@jslavin)
- Repository: https://github.com/HinodeXRT/xrtpy
- Documentation link: https://xrtpy.readthedocs.io/en/stable/
- PyPI link (if applicable): https://pypi.org/project/xrtpy/
- Continuous integration link(s): https://github.com/HinodeXRT/xrtpy/actions
- Link to Code of Conduct or page describing adopted Code of Conduct: TBA

## Description of Package

XRTpy is a Python package developed to facilitate the analysis of observations from the X-Ray Telescope (XRT) aboard the Hinode spacecraft. It provides specialized tools for processing high-resolution solar corona images and diagnosing coronal temperatures, supporting the research needs of the solar physics community.

## Editor Submission Checklist

- [x] Is the submission appropriate (compatible with the SunPy CoC)?
- [x] Does the project have an [appropriate license](https://opensource.org/licenses/)?
- [x] Is the project in an online version control system?
- [x] Does the project provide a Python interface?
- [x] Is the project on PyPI?
- [x] Is the project useful to the solar physics community?
- [x] Version that was reviewed (tag or commit hash) - v0.4.1

## Instructions to Reviewer

Please copy the following and select the ranking for each criteria, the full review criteria can be found here:

```
* Functionality           : General Package / Specialized Package / Not Relevant
* Integration             : Well integrated / Partially Integrated / Minimal Integration
* Documentation           : Extensive / Some / Little
* Testing                 : Excellent / Good / Needs Work
* Duplication             : None / Some / Major
* Community               : Excellent / Good / Needs Work
* Development Status      : Stable / Subject to Change / Low Activity / Needs Work
```

### PR DESCRIPTION
**Pull request by wtbarnes**
Fixes #431 
Fixes #414 

Should also add `xrtpy` do the docs dropdown, but I believe this involves modifying the theme.

### CODE REVIEW

**Comment by wtbarnes** [comment]
Hold up. Let me just add irispy-lmsal too

**Comment by nabobalis** [comment]
We will also need to patch the theme

File: affiliated.rst [hunk 1]
```diff
@@ -154,6 +154,27 @@ Current packages
...
+       |package_specialized| |integration_full| |docs_good| |tests_good| |duplication_none| |community_good| |dev_stc|
+
+       Version reviewed: `v0.4.1 <https://github.com/HinodeXRT/xrtpy/releases/tag/v0.4.1>`__
+
+   * - **irispy-lmsal**
+     - A Python package that provides the tools to read in and analyze data from the IRIS solar-observing satellite.
+
+       `Documentation <https://irispy-lmsal.readthedocs.io>`__, `Source code <https://github.com/LM-SAL/irispy-lmsal>`__
+
+       **Maintainer**: `Nabil Freij`_
```
Comment (nabobalis) [inline affiliated.rst:173]: Can I be anonymous?

**Review by nabobalis — APPROVED** [review]

File: affiliated.rst (hunk 1 above)
Comment (Cadair) [reply affiliated.rst:173]: Is there a group we could list (with some contact info) if you don't want it to be you personally?

**Review by Cadair — APPROVED** [review]

File: affiliated.rst (hunk 1 above)
Comment (nabobalis) [reply affiliated.rst:173]: Could just say Lmsal and link to the website?

Comment (Cadair) [reply affiliated.rst:173]: I'd rather give some concrete contact details or it just looks like it's not maintained.

Comment (wtbarnes) [reply affiliated.rst:173]: Is there an IRIS email address?

Comment (nabobalis) [reply affiliated.rst:173]: I am not aware of one. But I can ask at work.

**Comment by nabobalis** [comment]
At some point we need to work out a better way to arrange the dropdown.

**Pull request merged**